NuRadioReco/utilities/muon_flux.py

```diff
@@ -0,0 +1,154 @@
+import scipy.constants
+import numpy as np
+import json
+import scipy.signal
+from NuRadioReco.utilities import units, io_utilities
+import astropy
+import scipy.interpolate as interpolate
+from scipy.integrate import quad
+from scipy.integrate import dblquad
+
+plot_flux = False
```

Comment:
Maybe rather than having a `plot_flux` variable, you can protect the script with an `if __name__ == "__main__"` line. Then the plotting part will only run if you call `python muon_flux.py` as a script. Optionally, you could also add a docstring to the script to explain what it plots here.

Reply:
Thanks, nice idea. I implemented it :)